features/modified_execution_paths/class_method_call_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../feature_helper'
+
+describe 'Changing source file' do
+  subject(:forecast) do
+    Crystalball.foresee(workdir: root, map_path: root.join('execution_map.yml')) do |predictor|
+      predictor.use Crystalball::Predictor::ModifiedExecutionPaths.new
+    end
+  end
+  include_context 'simple git repository'
+
+  it 'adds class2 example when class1 changes because of class method call' do
+    change class1_path
```

Comment:
This is not enough to test it. This case will be handled as well by coverage strategy

Reply:
This is still not enough

Reply:
@EPecherkin I changed `Class1.foo` to be an empty method and confirmed it is not caught by coverage or any of the other strategies.